ic-http/src/mock.rs

```diff
@@ -91,10 +88,11 @@ pub(crate) async fn http_request(
     }
 
     // Return the error if one is specified.
```

Comment:
I guess this comment might need a small adjustment? Or just move it in the case where we have an error in the match?